src/main/java/appeng/hotkeys/LocatingServicesImpl.java

```diff
@@ -0,0 +1,59 @@
+package appeng.hotkeys;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+import appeng.api.hotkeys.LocatingService;
+import appeng.api.hotkeys.LocatingServices;
+import appeng.core.definitions.AEItems;
+import appeng.core.definitions.ItemDefinition;
+import appeng.items.tools.powered.PortableCellItem;
+
+/**
+ * registers {@link LocatingService}
+ */
+public class LocatingServicesImpl {
+    public static final HashMap<String, List<LocatingService>> REGISTRY = new HashMap<>();
+
+    static {
+        register(
+                new InventoryLocatingService(AEItems.WIRELESS_TERMINAL.asItem(),
+                        (player, i) -> AEItems.WIRELESS_TERMINAL.asItem().openFromInventory(player, i)),
+                "wireless_terminal");
+        register(
+                new InventoryLocatingService(AEItems.WIRELESS_CRAFTING_TERMINAL.asItem(),
+                        (player, i) -> AEItems.WIRELESS_CRAFTING_TERMINAL.asItem().openFromInventory(player, i)),
+                "wireless_crafting_terminal");
```

Comment:
Shouldn't this one be under `wireless_terminal`? Since the wireless crafting terminal should just supersede the regular wireless terminal.